pkgs/development/tools/misc/stlink/default.nix

```diff
@@ -1,4 +1,13 @@
-{ lib, stdenv, fetchFromGitHub, cmake, libusb1 }:
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, libusb1
+, gtk3
+, pkgconfig
+, wrapGAppsHook
+, withGUI ? false
```

Comment:
Can we remove this option and enable it by default?

Reply:
```
withGUI = false;
/nix/store/1y8pw2m3mq387gk3yx2s77d5rpiqjvms-stlink-1.7.0	   87806040
withGUI = true;
/nix/store/jgn2i9v5h5xqj15hkfhrdc9hxbc969kp-stlink-1.7.0	  336654944
```

Reply:
thats quite a bit bigger.

Reply:
My reasoning for the option was the following. This package is a collection of CLI tools, one of which happens to have a rudimentary GUI. Since it's easy to do with nix, it's nice to offer the GUI as an option, but there are probably cases where you do not want to pull in GTK3, just for a microcontroller programmer.

I think enabling the option by default is reasonable since most users will already have GTK3 dependencies. Would that work @SuperSandro2000?